workspaces/spectacle-shared/src/local-cli/client.ts

```diff
@@ -21,6 +21,8 @@ import {
   IValueAffordanceSerializationWithCounterGroupedByDiffHash,
 } from '@useoptic/cli-shared/build/diffs/initial-types';
 
+let throttlerPromiseSingleton = Promise.resolve();
```

Comment:
this should be class-local state, but I'd still recommend using something like bottleneck